assets/js/components/dashboard-sharing/ModuleRecoveryAlert/index.js

```diff
@@ -208,15 +311,48 @@ export default function ModuleRecoveryAlert() {
 				{ Object.keys( recoveryErrors ).length > 0 && (
 					<Errors recoveryErrors={ recoveryErrors } />
 				) }
+				<CTALink
+					id={ id }
+					ctaLabel={ __( 'Recover', 'google-site-kit' ) }
+					onCTAClick={ handleRecoverModules }
+					isSaving={ recoveringModules }
+				/>
+				<CTALinkSubtle
+					id={ id }
```

Comment:
Any reason why we aren't using the standard `<Dismiss>` component in all of these cases? Then we don't need to pass `onCTAClick` to `dismissNotification`.

Actually, we probably can use `<ActionsCTALinkDismiss>` as they provide a combination of `CTALink` and `Dismiss`.

Reply:
I've refactored to use this component. One thing to note is that, because we can't vary the `dismissExpires` independently between the CTA and Dismiss CTA the the notification won't show again if the error recurs within one day.